### PR TITLE
2023-01-16 Dependency bumps

### PR DESCRIPTION
**Pull request by stevejalim**
Resolves #12601 Bump boto3 from 1.26.45 to 1.26.50 in /requirements 
Resolves #12599 Bump twilio from 7.16.0 to 7.16.1 in /requirements 
Resolves #12598 Bump sentry-sdk from 1.12.1 to 1.13.0 in /requirements 
Resolves #12579 Bump phonenumberslite from 8.13.3 to 8.13.4 in /requirements 
Resolves #12597 Bump requests from 2.28.1 to 2.28.2 in /requirements

### CODE REVIEW

**Comment by stevejalim** [comment]
@alexgibson This can wait till after today's release for a merge - better to leave it on Dev for a day or two than ship this today, I think

**Review by alexgibson — APPROVED** [review]

**Pull request merged**
